### ISSUE TITLE
Création de chemin lors d'un changement de page ?

### ISSUE THREAD

**Issue by ysalmon** [ClementGre/PDF4Teachers#134]
### Prerequisite

- [X] I checked the bug was not already reported [here](https://github.com/clementgre/PDF4Teachers/issues?q=is%3Aissue+milestone%3A%22Release+1.3.2%22+-label%3A%22user+Issue%22+-label%3Aduplicate+-label%3Adocumentation+-label%3Aenhancement+-label%3A%22good+first+issue%22+-label%3A%22help+wanted%22+-label%3Aquestion)
- [X] I checked the bug is not an existing user issue [here](https://github.com/clementgre/PDF4Teachers/issues?q=is%3Aissue+label%3A%22user+issue%22+)

### Describe the bug

J'ai commencé à utiliser la nouvelle fonctionnalité de correction à main levée en continu. C'est très pratique !

J'observe cependant un phénomène curieux : lorsque ce mode est actif, le changement de page semble provoquer quelque chose.

### To Reproduce

1. Ouvrir un document de plusieurs pages.
2. Passer en mode dessin à main levée (Ctrl+D)
3. faire des choses
4.  Sauvegarder l'édition (Ctrl+S)
5. Sans sortir du mode dessin, placer le curseur de la souris sur une autre page (soit en le déplaçant, soit en faisant défiler le document avec la molette de la souris), ne pas cliquer.

Dans la barre de titre, le nom du document est affecté d'une étoile, ce qui indique une modification depuis la dernière sauvegarde.

Si on fait l'étape 5 après avoir quitté le mode dessin, cette étoile n'apparait pas.

### Desktop

kubuntu 22.04

**Comment by ClementGre** [comment]
Bonjour,

cela vient du fonctionnement même de cette fonctionnalité.
En fait, lorsque la souris survole une page alors que le mode dessin est activé sur une autre page, un nouvel élément vectoriel est créé sur la nouvelle page survolée, et il est automatiquement passé en mode dessin.

C'est ainsi que le document est marqué comme édité : un nouvel élément a été créé. Bien sûr, si le mode dessin est quitté alors que l'élément vectoriel est vide, il sera automatiquement supprimé.

**Comment by ysalmon** [comment]
Mais donc ça veut dire que si on défile de façon répétée des pages, on encombre l'édition avec des éléments vectoriels vides ?

**Comment by ClementGre** [comment]
Non, car si le mode dessin est quitté alors que l'élément vectoriel est vide, l'élément sera automatiquement supprimé, justement pour éviter ce genre de problème. Il est inutile de garder un élément qui est "vide".
